resolwe/test/testcases/process.py

```diff
@@ -240,8 +240,23 @@ def tearDown(self):
 
         super().tearDown()
 
+        # Check test outcome to prevent failing the test twice.
+        # Adapted from: https://stackoverflow.com/a/39606065
+        def list2reason(exc_list):
+            """Error reason conversion helper."""
+            if exc_list and exc_list[-1][0] is self:
+                return exc_list[-1][1]
+
+        if hasattr(self, '_outcome'):  # Python 3.4+
+            result = self.defaultTestResult()  # these 2 methods have no side effects
+            self._feedErrorsToResult(result, self._outcome.errors)
+        else:  # Python 3.2 - 3.3 or 3.0 - 3.1 and 2.7
```

Comment:
We could easily drop this part as we only support Python 3.4+?

Reply:
👍

Reply:
Yes, drop this part.